File: templates/nodejs/v/20/src/task.js
async function main() {
  try {
    const runtimeEnv = process.env.RUNTIME_ENV || 'unknown';
    const msg = `Hello from ${runtimeEnv}!`;
    console.log(msg);

    // Print all command-line arguments
    if (process.argv.length > 2) {
      console.log('Received the following arguments:');
      process.argv.slice(2).forEach((arg, index) => {
        console.log(`Argument ${index + 1}: ${arg}`);
      });
    } else {
      console.log('No arguments were passed.');
    }
    
    return 0;

  } catch (error) {
    console.error("Error processing environment variables:", error);
    return 1;
  }
}

process.exit(await main());

File: templates/python/v/3.13/src/task.py
import os
import sys


def main():
    runtime_env = os.getenv('RUNTIME_ENV', 'unknown')
    msg = f"Hello from {runtime_env}!"
    print(msg)

    if len(sys.argv) > 1:
        print('Received the following arguments:')
        for index, arg in enumerate(sys.argv[1:], start=1):
            print(f"Argument {index}: {arg}")
    else:
        print('No arguments were passed.')

    return 0


if __name__ == '__main__':
    main()
